targets/chrome-common

```diff
@@ -26,26 +26,30 @@ REQUIRES='x11'
 . "${TARGETSDIR:="$PWD"}/common"
 
 ### Append to prepare.sh:
-# Only download if it's not already installed, since it auto-updates
-pkg="google-chrome-$CHANNEL"
-if [ -n "`list_uninstalled '' "$pkg"`" ]; then
-    # Chrome needs hicolor-icon-theme to configure/install
-    install hicolor-icon-theme
+if [ "${DISTROAKA:-"$DISTRO"}" = 'arch' ]; then
+    install aur:google-chrome$CHANNEL
```

Comment:
Note: Instead of installing chromium automatically, just like we do on ARM (because there is no alternative), the target fails with `error 99`, so that the user can make an informed choice. (ca20031ab8ead740a7df73db97abe215ef53e7a7)